### PR TITLE
Curtis items page / wishlist is done for now

### PR DESCRIPTION
**Pull request by Curtisdak**

### CODE REVIEW

**Review by Micka90 — APPROVED** [review]

File: client/src/components/Card/Card.jsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+import { FaHeart } from "react-icons/fa";
 import { SlOptionsVertical } from "react-icons/sl";
 import { MdOutlineEuroSymbol } from "react-icons/md";
+import { toast } from "react-toastify";
 import PropTypes from "prop-types";
 import { useEffect, useState } from "react";
 
-function Card({ product, setShowInput,cart, setCart }) {
+function Card({ product, setShowInput, cart, setCart , favorites, setFavorites  }) {
   const port = import.meta.env.VITE_API_URL;
```
Comment (Alexandre78R) [inline client/src/components/Card/Card.jsx]: rename urlApi

File: client/src/components/Modal/ModalFav/ModalFav.jsx [hunk 2]
```diff
@@ -1,41 +1,47 @@
...
-  },
-  {
-    id: 3,
-    img: "src/assets/images/categoryPhotos/earrings_1.webp",
-    name: "Collier argent",
-    details: "Marque Tiffany & Co, sertie de 40 diamants",
-    price: "€3500",
-  },
-];
+  const port = import.meta.env.VITE_API_URL;
```
Comment (Alexandre78R) [inline client/src/components/Modal/ModalFav/ModalFav.jsx]: rename urlApi

File: client/src/components/Modal/ModalNav/ModalNav.jsx [hunk 3]
```diff
@@ -1,43 +1,52 @@
...
-    img: "src/assets/images/categoryPhotos/earrings.jpg",
-    name: "Collier argent",
-    details: "Marque Tiffany & Co, sertie de 40 diamants",
-    price: "€3500",
-  },
-];
+function ModalNav({ setModalNav, setFavorite }) {
+  const [modalNavigation, setModalNavigation] = useState([]);
+
+  const port = import.meta.env.VITE_API_URL;
```
Comment (Alexandre78R) [inline client/src/components/Modal/ModalNav/ModalNav.jsx]: rename urlApi

File: server/app/controllers/wishlistAction.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+const tables = require("../../database/tables");
+
+const addWishlist = async (req, res, next) => {
+  try {
+    const like = await tables.wishlist.create(req.body);
+    res.status(200).json(like);
```
Comment (Alexandre78R) [inline server/app/controllers/wishlistAction.js]: change port 201

**Review by Alexandre78R — CHANGES_REQUESTED** [review]

**Review by VinchC — APPROVED** [review]

**Pull request merged**